cms/templates/howitworks.html

```diff
@@ -14,8 +16,8 @@
 <div class="wrapper-content-header wrapper">
   <section class="content content-header">
     <header>
-      <h1><span class="wrapper-text-welcome">${_("Welcome to {studio_name}").format(
-        studio_name=u'</span><span class="logo">{studio_name}</span>'.format(
+      <h1><span class="wrapper-text-welcome">${Text(_("Welcome to {studio_name}")).format(
+        studio_name=HTML(u'</span><span class="logo">{studio_name}</span>').format(
```

Comment:
@mushtaqak how do I render this template?  I don't see any way to load this template in Studio.  Its a little hard to see what the CSS is doing here without actually rendering it, so I'm a little hesitant to make a recommendation.  I'm a fan of keeping it simple, so I'm favoring the first solution @mushtaqak recommends, but I don't really know what that logo span is doing.  I imagine its loading the logo as a background image?  If so, that is a really bad idea and I would recommend against that approach.  A logo is content and shouldn't be included as a CSS bg.